### DISCUSSION TITLE
quasar dev encounter await unexpected reserved word syntaxerror

### DISCUSSION THREAD

**Discussion by tuantuanzi-code** [quasarframework/quasar#17009] [General - Components / Directives / etc]
I have install @quasar/cli@2.4.0, but quasar can not work due to quasar.js line 25,   const { default: appPaths } = await import('../lib/app-paths.js')
                                ^^^^^

SyntaxError: Unexpected reserved word

![image](https://github.com/quasarframework/quasar/assets/82799087/0c3d7ed7-1229-437f-aec0-833369db0a92)

**Comment by yusufkandemir** [comment]
Which Node version you are using? (`node -v`)

**Reply by tuantuanzi-code** [reply]
hi, I have removed all install and reinstall, but now meet dev is unknown cmd issue, I have tried thishttps://github.com/quasarframework/quasar/issues/5970, but failed
I am working in windows, node is v16.14.0, npm is 7.20.3, quasar is @quasar/cli v2.4.0.
I follow below steps:
·  Install node.js v14.14.0, .msi is for windows at https://nodejs.org/zh-cn/download/releases/ 
·  Install quasar: npm install -g @quasar/cli, [Quasar CLI | Quasar Framework](https://quasar.dev/start/quasar-cli)

·  open an existed project, cd client， remove folder: node_modules
·  npm install
·  npm install -g node-sass --force
·  then run 'quasar dev', and reports quasar/cli need node version higher than V16.0.0, so I unintall v14.14.0, install v16.14.0 instead,
  then run 'quasar dev' again, reports dev is unknown cmd, I have run 'quasar upgrade -i' but still fail to run dev

the existed project is build almost 3 years ago, I am not sure is ther any version not match or something, could you please help? Thanks.

seems the existed project issue, if I run under a new init project folder, it works well.
is it related to quasar version? seems the existed project uses ' "quasar": "^1.22.10" after check package.json, but quasar info reports @quasar/cli - 2.4.0.

**Reply by yusufkandemir** [reply]
`@quasar/cli` requires Node v16.

`quasar -v` outputs the `@quasar/app*` package if executed in the project, and the global `@quasar/cli` versions if executed outside a project. As the output is `2.4.0`(_latest `@quasar/cli` version_), it's not picking up the project.

I would suggest removing `node_modules` again and installing everything using Node v16. If it doesn't work, remove both `node_modules` and `package-lock.json`, then install packages.

**Reply by tuantuanzi-code** [reply]
Hi, thanks for your suggestion. 
I have tried, but now quasar dev met another problem, it will report :
 Error: Node Sass does not yet support your current environment: Windows 64-bit with Unsupported runtime (93)
  For more information on which environments are supported please see:
  https://github.com/sass/node-sass/releases/tag/v4.14.1

but seems windows node sass only support node 14, do you have any idea how to fix it?

**Reply by yusufkandemir** [reply]
Then, you should use Node v14, but either use global Quasar CLI v1, or don't use it all. You can add a new script to `package.json > scripts`: `"dev": "quasar dev"`, then run it with `npm run dev`. That way, it will use `@quasar/app` which supports Node v14, but not `@quasar/cli` which doesn't support it.

**Reply by tuantuanzi-code** [reply]
thanks a lot. somehow I have run successfully if I just ignore the message and do not try  npm audit fix --force after npm install.

but seems my way is in a risk. do you suggest me to migrate my code to quasar v2 with following this guide ?https://quasar.dev/start/upgrade-guide#migrate-to-v2-from-v1

**Reply by yusufkandemir** [reply]
Yes, v1 is dead. You should definitely migrate to v2, whenever you can.
